fiberplane-models/src/notebooks/front_matter.rs

```diff
@@ -647,3 +652,110 @@ impl TryFrom<serde_json::Value> for FrontMatterPagerDutyIncident {
         })
     }
 }
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize, TypedBuilder)]
+#[cfg_attr(
+    feature = "fp-bindgen",
+    derive(Serializable),
+    fp(rust_module = "fiberplane_models::notebooks::front_matter")
+)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct FrontMatterGitHubPullRequest {
+    /// HTML url of this GitHub pull request. This is the only required value when creating
+    /// this front matter schema. Every other field is marked as optional and will be automatically
+    /// populated by the backend whenever front matter values get updated or an incoming GitHub
+    /// webhook got handled.
+    pub html_url: String,
+
+    /// Global unique ID of the GitHub pull request. This gets used to find this very front matter
+    /// within all notebooks whenever a webhook from GitHub gets handled. GitHub assigns this ID.
+    #[builder(setter())]
+    pub id: u64,
+
+    /// The owner of the repository where this pull request was created on
+    #[builder(setter(into))]
+    pub repo_owner: String,
+
+    /// The name of the repository where this pull request was created on
+    #[builder(setter(into))]
+    pub repo_name: String,
+
+    /// The pull request number within this repository. Please note that GitHub
+    /// treats pull request and issue numbers as the same.
+    #[builder(setter())]
+    pub number: u64,
+
+    /// Title of the pull request
+    #[builder(setter(into))]
+    pub title: String,
+
+    /// Branch name of this PR
+    #[builder(setter(into))]
+    pub branch: String,
+
+    /// Amount of commits in this PR
+    #[builder(setter())]
+    pub commits: u64,
+
+    /// Creator of the pull request
+    #[builder(setter(into))]
+    pub author: String,
+
+    /// GitHub Avatar URL of the author
+    #[builder(setter(into))]
+    pub author_avatar_url: String,
+
+    /// Assignee of this pull request
+    #[builder(setter(into))]
+    pub assignee: Option<String>,
+
+    /// GitHub avatar URL of the assignee
+    #[builder(setter(into))]
+    pub assignee_avatar_url: Option<String>,
+
+    /// Labels attached to this PR
+    #[builder(setter())]
+    pub labels: Vec<String>,
+
+    /// Reviewers requested for this PR
+    #[builder(setter())]
+    pub reviewers: Vec<String>,
+
+    /// State of the pull request
+    #[builder(setter(into))]
+    pub state: String,
+
+    /// Whenever the pull request is a draft
+    #[builder(setter())]
+    pub draft: bool,
+
+    /// Whenever the pull request was merged
+    #[builder(setter())]
+    pub merged: bool,
+
+    /// Timestamp when this pull request was created
+    #[builder(setter(into))]
+    pub created_at: Timestamp,
+
+    /// Timestamp of the last update received by Fiberplane to this pull request. May be outdated
+    /// if there are changes that the GitHub webhook has not yet sent out
+    #[builder(setter(into))]
+    pub last_updated: Timestamp,
```

Comment:
We should make this match our conventions

```suggestion
    pub updated_at: Timestamp,
```